Src/ILGPU/Runtime/ExchangeBufferExtensions.cs

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
ISSUE: `ExchangeBufferExtensions.cs` is a generated file. Please remove it from the commit, and add it to `.gitignore`

Reply:
I thought I hadn't committed the generated version but oh well, I wasn't aware I had to add it to the .gitignore. I'll do that right away.